control/trajectory_follower/src/mpc_lateral_controller.cpp

```diff
@@ -79,13 +79,13 @@ MpcLateralController::MpcLateralController(rclcpp::Node & node) : node_{&node}
   m_new_traj_end_dist = node_->declare_parameter<float64_t>("new_traj_end_dist");            // [m]
 
   /* mpc parameters */
-  const float64_t steer_lim_deg = node_->declare_parameter<float64_t>("steer_lim_deg");
+  const auto vehicle_info = vehicle_info_util::VehicleInfoUtil(*node_).getVehicleInfo();
+  const float64_t wheelbase = vehicle_info.wheel_base_m;
+  const float64_t steer_lim_deg = vehicle_info.max_steer_angle_rad;
```

Comment:
need unit conversion: rad to deg.

You can get rid of the `steer_lim_deg` and substitute the arg directly.
```
m_mpc.m_steer_lim = vehicle_info.max_steer_angle_rad;
```

Reply:
(Sorry, this comment was pending for a long time :bow: )

Reply:
ok